backend/src/apps/mentorship/api/internal/nodes/mentor.py

```
@@ -1,14 +1,18 @@
 """GraphQL node for Mentor model."""
 
+from typing import TYPE_CHECKING
+
 import strawberry
 
+if TYPE_CHECKING:
+    from apps.github.models.user import User as GitHubUser
```

Ruff (0.15.20)

[warning] Move import `apps.github.models.user.User` out of type-checking block. Import is used for more than type hinting.

Move out of type-checking block

(TC004)